websites/S/Serendia Squad/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.0",
+  "author": {
+    "name": "BBQ",
+    "id": "622527559946731540"
+  },
+  "url": "serendiasquad.com",
+  "description": {
+    "tr": "The Serendia Squad's code sharing website.",
+	"az": "Serendia Squad'ın kod paylaşım saytı.",
+    "en": "Serendia Squad'ın kod paylaşım sitesi.",
+	"fr": "Site de partage de code de Serendia Squad's."
+  },
```

Comment:
i think you kinda facked up here....

Reply:
azerbaijani description is wrong

Reply:
they all are

Reply:
> they all are

sorry its my mistake i uploaded old files 😂